packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -918,10 +946,12 @@ class _PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
     if (items.isNotEmpty) {
       showMenu<T>(
         context: context,
-        elevation: widget.elevation,
+        elevation: widget.elevation ?? popupMenuEntryTheme.elevation ?? 8.0,
```

Comment:
I see this logic (for elevation, shape, and color) repeated in 2 spots, does it have to be?

Reply:
Done (The paths are essentially combined into 1 now!)